packages/cli/src/__fixtures__/test-project/packages/app/package.json

```diff
@@ -0,0 +1,82 @@
+{
+  "name": "app",
```

Comment:
Remove everything here that isn't required to make the tests pass or adds valuable context